share/spice/spotify/spotify.js

```diff
@@ -0,0 +1,38 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_spotify = function(api_result){
+
+        var query = DDG.get_query();
+        query = query.replace(/spotify/, ""); //replace trigger words from query
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.tracks.items.length === 0) {
+            return Spice.failed('spotify');
+        }
+
+        // Render the response
+        Spice.add({
+            id: "spotify",
+
+            // Customize these properties
+            name: "Spotify",
+            data: api_result.tracks.items,
+            meta: {
+                sourceName: "Spotify.com",
+                sourceUrl: 'https://api.spotify.com/v1/search?q=' + query.trim() + '&type=track',
```

Comment:
Sure, I'm going to change it. The page should ideally be a public one where the user doesn't need to log in. But until Spotify implements it from their end, I will point to `https://play.spotify.com/search/<query>`.